pkg/apis/options/providers.go

```diff
@@ -209,16 +209,16 @@ type OIDCOptions struct {
 	IssuerURL string `json:"issuerURL,omitempty"`
 	// InsecureAllowUnverifiedEmail prevents failures if an email address in an id_token is not verified
 	// default set to 'false'
-	InsecureAllowUnverifiedEmail bool `json:"insecureAllowUnverifiedEmail,omitempty"`
```

Comment:
Why no longer omitempty?

Reply:
I decided to not omit any booleans. To ensure that the conversion flow from toml to yaml explicitly mentions all boolean values.